python/foglamp/services/core/server.py

```diff
@@ -118,6 +142,21 @@ def __start_storage(host, m_port):
             # callback with args
         loop.call_soon(cls.__start_storage, cls._host, cls.core_management_port)
 
+    @classmethod
+    async def _get_storage_client(cls):
+        storage_service = None
```

Comment:
I think its coming from Scheduler; and should be done ONCE only (keep DRY) on startup; as you are putting this on top of start schedule

Purpose was to make sure, Scheduler starts only when storage is available; and `if _ENV != 'TEST':` was used there for  run time ENV var export to run  test `tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py` with  a comment Execute this command: `FOGLAMP_ENV=TEST pytest -s -vv`

Reply:
I needed a way for server to obtain a storage client. How else could I do it?

Reply:
I think we just need to put this def at common place instead and do polling once.

Reply:
@amarendra-dianomic could you please help here? 1) ENV VAR 2) SCRIPTS_DIR and FOGLAMP_ROOT DIR via env var 3) get storage client and use of that with pytest